tuner/tuner/dispatch_parser.py

```diff
@@ -11,6 +11,7 @@
 import re
```

Comment:
The changes here are basically just calling the op matchers to get the root ops.